ui/package.json

```diff
@@ -3,8 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "typescript": ">=1.5.0-beta",
+    "chai": "^3.2.0",
```

Comment:
yeah, we should bump all these versions then. phantomjs in particular has weird bugs pre 2.0